scripts/travis/main.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck disable=SC1090
+source "$SCRIPT_DIR/_includes.sh";
+
+fold_start "yarn-install"
+retry 3 yarn install
+fold_end
+
+fold_start "yarn-build"
+retry 3 yarn build
```

Comment:
Let's not retry the build. For the installation (which depends on network) this makes sense, but when the build failed there is no point in blocking further CI time